ontology/ngbo.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: ontology_detail
+id: ngbo
+title: Next Generation Biobanking Ontology
+contact:
+  email: dal.alghamdi92@gmail.com
+  github: dalalghamdi
+  label: Dalia Alghamdi
+  orcid: 0000-0002-2801-0767
+description: 'The Next Generation Biobanking Ontology (NGBO) is an open application ontology representing contextual data about omics digital assets in biobank. The ontology focuses on capturing the information about three main activities: wet bench analysis used to generate omics data, bioinformatics analysis used to analyze and interpret data, and data management. The scope of the ontology includes real-world physical objects such as specimens, computerized objects such as datasets, and planned processes and related specifications. The intended end-users are biobanking organizations, data collectors, omics researchers, and decision-makers. 
```

Comment:
description should only be one sentence - please move the rest of this to the main body of the markdown file